packages/adp-tooling/src/base/abap/manifest-service.ts

```diff
@@ -31,12 +37,18 @@ export class ManifestService {
      *
      * @param adpConfig - The ADP preview configuration.
      * @param logger - The logger instance.
+     * @param requestOptions - Additional AxiosRequestOptions
```

Comment:
The param description could be adjusted to be more descriptive: _Optional configuration options for the Axios request, such as headers, timeout, and base URL._

Reply:
Fixed in: ea1b35309b367c193c5bbd833d96cdd9613c5293